boards/nxp/imx943_evk/imx943_evk_mimx94398_m33_defconfig

```diff
@@ -21,3 +21,7 @@ CONFIG_ARM_SCMI_NXP_VENDOR_EXTENSIONS=y
 
 # kernel-related configurations
 CONFIG_XIP=y
+
+# i2c shell-related configurations
+CONFIG_I2C=y
+CONFIG_I2C_SHELL=y
```

Comment:
I doubt you want to make this the defaults for anyone using this board?

Reply:
> I doubt you want to make this the defaults for anyone using this board?

Yes, you're right.

Reply:
It looks still there, is it your intention that every build for every app with this board should include an i2c shell?

Reply:
> It looks still there, is it your intention that every build for every app with this board should include an i2c shell?

No, it's not my intention. Need to enable these options(CONFIG_I2C, CONFIG_I2C_SHELL) in the test case(samples/subsys/shell/shell) manually. So i will remove these options in boards/nxp/imx943_evk/imx943_evk_mimx94398_m33_defconfig